drivers/SmartThings/matter-appliance/src/matter-refrigerator/init.lua

```diff
@@ -122,9 +150,7 @@ end
 
 local function setpoint_limit_handler(limit_field)
   return function(driver, device, ib, response)
-    local tn_eps = embedded_cluster_utils.get_endpoints(device,clusters.TemperatureControl.ID, {feature_bitmap = clusters.TemperatureControl.types.Feature.TEMPERATURE_NUMBER})
-    if #tn_eps == 0 then
-      device.log.warn_with({ hub_logs = true }, string.format("Device does not support TEMPERATURE_NUMBER feature"))
+    if not supports_temperature_number_endpoint(device, ib.endpoint_id) then
```

Comment:
I don't think this function is needed, because the `setpoint_limit_handler` function would only be called in the case that the endpoint supports the attribute `MinTemperature` or `MaxTemperature`:
```
    attr = {
      [clusters.TemperatureControl.ID] = {
        [clusters.TemperatureControl.attributes.TemperatureSetpoint.ID] = temperature_setpoint_attr_handler,
        [clusters.TemperatureControl.attributes.MinTemperature.ID] = setpoint_limit_handler(setpoint_limit_device_field.MIN_TEMP),
        [clusters.TemperatureControl.attributes.MaxTemperature.ID] = setpoint_limit_handler(setpoint_limit_device_field.MAX_TEMP),
```

So I think we can assume that value `ib.endpoint` does have temperatureNumber suppport.

Reply:
It is the same with the following [comment](https://github.com/SmartThingsCommunity/SmartThingsEdgeDrivers/pull/1500/files#r1694798795)